src/testing.rs

```diff
@@ -442,6 +522,22 @@ impl LogBuilder {
     }
 }
 
+/// Simple storage solidity contract that stores and retrieves two numbers
+///
+/// contract Storage {
+///     uint256 number1 = 1024;
+///     uint256 number2 = 115792089237316195423570985008687907853269984665640564039457584007913129639935;
```

Comment:
nit: Might be worth mentioning that this is `U256::MAX` since we use that value for our tests?